src/test/java/mat/server/util/MeasureUtilityTest.java

```diff
@@ -17,7 +17,8 @@ public void testVersionTextIfDraftable() {
         assertEquals(MeasureUtility.getVersionText(version, isDraft), "Draft based on v4.2");
         assertEquals(MeasureUtility.getVersionTextWithRevisionNumber(version, revisionNumber, isDraft), "Draft v4.2.005");
         assertEquals(MeasureUtility.formatVersionText(revisionNumber, version), "4.2.005");
-        assertEquals(MeasureUtility.formatVersionTextForIncludeLibraries(version + "." + revisionNumber), "4.2.005");
+        assertEquals(MeasureUtility.formatVersionText(version + "." + revisionNumber), "4.2.005");
+        assertEquals(MeasureUtility.formatVersionText(version), "4.2");
```

Comment:
JUnit asserts follow a pattern of `assert(expected, actual)`. Looks like the arguments are reversed in these asserts.

Reply:
my bad will update them